google/cloud/security/inventory/inventory_loader.py

```diff
@@ -172,8 +171,7 @@ def _run_pipelines(pipelines):
             pipeline.run()
             pipeline.status = 'SUCCESS'
             LOGGER.info('Finished running %s', pipeline.__class__.__name__)
-        except (api_errors.ApiInitializationError,
-                inventory_errors.LoadDataPipelineError) as e:
+        except Exception as e:
```

Comment:
I think that this blanket exception should be in addition to the exceptions that we can catch:
```
except (api_errors.ApiInitializationError, inventory_errors.LoadDataPipelineError) as e:)
   handle as before
except Exception as e:
  log unknown failure as occurred
```